cves/CVE-2019-9721.yml

```diff
@@ -342,7 +350,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: The mistakes that were made when developing this was a lack of understanding of the different complex file inputs, and how the can be used by an attacker.
```

Comment:
Possibly split into multi-line again. Some of your responses are multi-line and some aren't.